<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="7" passed="6" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2024-03-12T11:38:10 IST" name="Suite" finished-at="2024-03-12T11:38:58 IST" duration-ms="48504">
    <groups>
    </groups>
    <test started-at="2024-03-12T11:38:10 IST" name="Test" finished-at="2024-03-12T11:38:58 IST" duration-ms="48504">
      <class name="com.scenario.automate.HandleAlertTests">
        <test-method is-config="true" signature="loadEverything()[pri:0, instance:com.scenario.automate.HandleAlertTests@3835c46]" started-at="2024-03-12T11:38:10 IST" name="loadEverything" finished-at="2024-03-12T11:38:22 IST" duration-ms="12603" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loadEverything -->
        <test-method signature="cancelAlertTest()[pri:0, instance:com.scenario.automate.HandleAlertTests@3835c46]" started-at="2024-03-12T11:38:22 IST" name="cancelAlertTest" finished-at="2024-03-12T11:38:33 IST" duration-ms="10450" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- cancelAlertTest -->
        <test-method signature="handleAlertTest()[pri:0, instance:com.scenario.automate.HandleAlertTests@3835c46]" started-at="2024-03-12T11:38:33 IST" name="handleAlertTest" finished-at="2024-03-12T11:38:37 IST" duration-ms="4330" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- handleAlertTest -->
        <test-method signature="restApiTestTwo()[pri:0, instance:com.scenario.automate.HandleAlertTests@3835c46]" started-at="2024-03-12T11:38:37 IST" name="restApiTestTwo" finished-at="2024-03-12T11:38:39 IST" duration-ms="2309" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- restApiTestTwo -->
        <test-method signature="sendAlertMessageTest()[pri:0, instance:com.scenario.automate.HandleAlertTests@3835c46]" started-at="2024-03-12T11:38:39 IST" name="sendAlertMessageTest" finished-at="2024-03-12T11:38:51 IST" duration-ms="12064" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- sendAlertMessageTest -->
        <test-method signature="testClosePopUp()[pri:0, instance:com.scenario.automate.HandleAlertTests@3835c46]" started-at="2024-03-12T11:38:51 IST" name="testClosePopUp" finished-at="2024-03-12T11:38:57 IST" duration-ms="5928" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [false] but found [true]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:1067)
at org.testng.Assert.assertEquals(Assert.java:1031)
at org.testng.Assert.assertEquals(Assert.java:1077)
at com.scenario.automate.HandleAlertTests.testClosePopUp(HandleAlertTests.java:124)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:228)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:961)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:201)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:819)
at org.testng.TestRunner.run(TestRunner.java:619)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1228)
at org.testng.TestNG.runSuites(TestNG.java:1134)
at org.testng.TestNG.run(TestNG.java:1101)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:308)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:71)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:113)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:385)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:507)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:495)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testClosePopUp -->
        <test-method is-config="true" signature="closeAll()[pri:0, instance:com.scenario.automate.HandleAlertTests@3835c46]" started-at="2024-03-12T11:38:58 IST" name="closeAll" finished-at="2024-03-12T11:38:58 IST" duration-ms="438" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closeAll -->
      </class> <!-- com.scenario.automate.HandleAlertTests -->
      <class name="com.scenario.automate.RestApiTest">
        <test-method is-config="true" signature="loadUp()[pri:0, instance:com.scenario.automate.RestApiTest@192d43ce]" started-at="2024-03-12T11:38:22 IST" name="loadUp" finished-at="2024-03-12T11:38:22 IST" duration-ms="1" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loadUp -->
        <test-method signature="restApiTestOne()[pri:1, instance:com.scenario.automate.RestApiTest@192d43ce]" started-at="2024-03-12T11:38:57 IST" name="restApiTestOne" finished-at="2024-03-12T11:38:58 IST" duration-ms="174" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- restApiTestOne -->
        <test-method signature="restApiTestTwo()[pri:2, instance:com.scenario.automate.RestApiTest@192d43ce]" started-at="2024-03-12T11:38:58 IST" name="restApiTestTwo" finished-at="2024-03-12T11:38:58 IST" duration-ms="160" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- restApiTestTwo -->
      </class> <!-- com.scenario.automate.RestApiTest -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
